.github/workflows/ci.yml

```diff
@@ -71,15 +71,15 @@ jobs:
             _boot
             rootfs-${{ matrix.target }}.tar
 
-  test-emu:
+  test-host:
     needs: build
-    name: run tests on emulators
+    name: run tests on ubuntu host
```

Comment:
coincidentally true, but not because there is ubuntu specified in the next line (but because phoenixrtos/devel:latest docker image uses ubuntu:20.04 as a base image)

Reply:
Thanks for review, this and the remaining changes above won't be necessary, if there are two separate targets in phoenix-rtos-poject. I've adjusted testing process for that, but I have to talk about it with Hubert. Now there is a test pr with changed zynq target name and this branch to run ci properly (https://github.com/phoenix-rtos/phoenix-rtos-project/pull/381).